src/askvm/resources/list/push.ts

```diff
@@ -0,0 +1,9 @@
+import { any, resource } from '../../lib';
+
+export const push = resource({
+  type: any,
+  async resolver(arr: Array<any>, value: any) {
+    arr.push(value);
```

Comment:
@mhagmajer - are we still immutable with this?

Reply:
Nope, we need to create a new array here. Also, it would be good to add a unit test that verifies immutability.